src/test/java/com/integrivideo/pages/ProjectListPage.java

```diff
@@ -21,9 +22,14 @@
     private WebElement addProjectButton;
 
     public void clickAddProject() {
+        Utils.scrollToElement(addProjectButton);
```

Comment:
If scrolls and waits for elements are really required - all of them should be moved to some wrapper class for buttons